samples/matter/light-bulb/readme.rst

```diff
@@ -0,0 +1,150 @@
+.. _matter-light-bulb:
+
+Matter Light bulb sample
+########################
+
+Overview
+********
+
+Sample for how to control a green dimmable light bulb endpoint and Matter SDK.
+Sample support BLE commisioning to Thread network.
+
+Requirements
+************
+
+- Alif Balletto Development Kit
+- Matter Thread Border router and Matter Controller
+- Pre provisioned Matter Light sample or 3rd party light switch
+
+Build Sample
+************
+
+Enable automatic start of the openthread stack
+
+.. code-block:: console
+
+   west build -p always -b alif_b1_dk_rtss_he samples/matter/light-bulb
+
+Light-bulb endpoint
+*********************
+
+Sample have generated ZAP and Matter file which enable with Matter Root endpoint 0 and OnOff Dimmable Light bulb endpoint 1.
+
+User interface
+**************
+
+Devkit use 2 leds for indicating Thread and BLE communication state
+
+PWM led (Green Led):
+- Dimmable Light controlled by Matter controller or binded On/Off light switch
+
+Blue LED:
+
+- Blinking slowly indicate that Matter BLE Advertisment is active for Matter Provisioning
+- Fast Blink (100ms) Means that BLE connection is active and Matter Provisioning is ongoing
+
+Red LED:
+
+- Fast Blinking (100ms) indicate Thread network is active for joining provisionmed newtwork
+- 1 Second constant Active Mode indicate That Matter controller subscribed is establis,ent or resumed
+- Single short active indicate Endpoint activation
+
+Sample support Balletto Devkit Joystick buttons
+
+Center Button `SW0`:
+
+- Hold over 3 seconds for doing Matter factoryreset. After Factory reset press DK normal Reset Button for activate BLE advertisment.
+
```

Comment:
Hold over 3 seconds for doing Matter factoryreset ->
Hold over 3 seconds for Matter factoryreset